introduction.md

```diff
@@ -1,3 +1,4 @@
 This is me . so now not empty 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?